lib/main_micro_alp.pm

```diff
@@ -46,12 +46,23 @@ sub load_config_tests {
 }
 
 sub load_boot_from_disk_tests {
-    # add additional image handling module for svirt workers
-    if (is_s390x()) {
-        loadtest 'installation/bootloader_start';
-    } elsif (is_vmware()) {
-        loadtest 'installation/bootloader_svirt';
-        loadtest 'installation/bootloader_uefi';
+
+    if (is_ipmi) {
+        loadtest 'installation/ipxe_install';
+        loadtest 'microos/install_image';
+    } elsif (is_pvm_hmc) {
+        # bootloader returns if BOOT_HDD_IMAGE is set, we need to unset it
+        set_var 'BOOT_HDD_IMAGE', '';
+        loadtest 'installation/bootloader';
+        loadtest 'microos/install_image';
+    } else {
+        # add additional image handling module for svirt workers
+        if (is_s390x()) {
+            loadtest 'installation/bootloader_start';
+        } elsif (is_vmware()) {
+            loadtest 'installation/bootloader_svirt';
+            loadtest 'installation/bootloader_uefi';
```

Comment:
I think one of those modules is a wrapper for the rest, I'd make sure they all support it, and remove the conditional parts

Reply:
The `tests/installation/bootloader_start.pm` is the wrapper, but was never used in image testing, rather it works fine with installations. It might be a nice future ticket, but it might be problematic to enforce such a change at this point.